query-service-impl/src/main/java/org/hypertrace/core/query/service/pinot/QueryRequestToPinotSQLConverter.java

```diff
@@ -320,10 +346,17 @@ private String convertExpressionToMapValueColumn(Expression expression) {
                       Value.newBuilder().setString(value.getValue().getStringArray(i)).build())
                   .build();
         }
+      } else if (value.getValue().getValueType() == ValueType.STRING) {
+        literals[1] =
```

Comment:
So literals `[0]` is null? We should return something better (but see my comment about operator support, may make this moot)

Reply:
Returning empty string instead